core-ui/src/components/Extensibility/migration.js

```diff
@@ -0,0 +1,60 @@
+import * as jp from 'jsonpath';
+import { cloneDeep } from 'lodash';
+
+const SUPPORTED_VERSIONS = [
+  { version: '0.4', latest: false },
+  { version: '0.5', latest: true },
```

Comment:
just a thought, maybe we could have 2 separate entries, like 
```
const SUPPORTED_VERSIONS = ['0.4', '0.5'];
const LATEST_VERSION = '0.5'
```
for me, it looks more readable